carball/json_parser/actor/player.py

```diff
@@ -17,49 +17,37 @@ def update(self, actor: dict, frame_number: int, time: float, delta: float) -> N
         }
         # Conditionally add ['team'] key to player_dict
         player_team = actor.get("Engine.PlayerReplicationInfo:Team", None)
-        if player_team is not None and player_team != -1:
+        # TODO this is unsigned in boxcars, remove 4294967295 when fixed
+        if player_team is not None and player_team != -1 and player_team != 4294967295:
```

Comment:
I believe this should be fixed as boxcars py was updated to the latest version of boxcars 👌

Reply:
Updated in 434d367